### PR TITLE
added histological quality

### LINKED ISSUE #342: Add cellular histological qualities
We have a set of terms in CL that should be obsoleted and replaced with PATO terms:

'acidophilic cytoplasm' http://purl.obolibrary.org/obo/CP_0000027
'basophilic cytoplasm' http://purl.obolibrary.org/obo/CP_0000028
'neutrophillic cytoplasm' http://purl.obolibrary.org/obo/CP_0000000
'polychromatophilic cytoplasm' http://purl.obolibrary.org/obo/CP_0000035

Suggested classification definition pattern for these terms:

cellular quality
. - cellular histological quality: A quality inhering in a cell by virtue of its capacity to be stained by specific histological dyes.
. . - cellular acidophilic* histological quality: A cellular quality inhering in a cell by virtue of its capacity to be stained by acidic dyes. Comment: In histopathology this is commonly indicated by pale-pink staining with Wright-Giemsa stain.

### PR DESCRIPTION
**Pull request by shawntanzk**
*** DO NOT MERGE - NOT READ***
Fixes #342
@dosumis this definitely needs checking - I had to move some stuff and I'm not sure how it will affect the modelling etc.

### CODE REVIEW

**Comment by shawntanzk** [comment]
related to https://github.com/obophenotype/cell-ontology/issues/572

**Comment by matentzn** [comment]
I have a rule to not be on content / modelling PRs :P Sorry.

**Comment by shawntanzk** [comment]
whoops, no worries, thought id put you cause of linked issue, but yeah, makes sense - will find another reviewer :)

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -15050,11 +15050,11 @@ creation_date: 2009-10-05T12:01:13Z
 [Term]
 id: PATO:0002094
 name: basophilic
-def: "An affinity inhering in a tissue constituent by virtue of the bearer exhibiting a molecular interaction for basic dyes under specific pH conditions." [PATOC:GVG]
+def: "An affinity inhering in an anatomical constituent by virtue of the bearer exhibiting a molecular interaction for basic dyes under specific pH conditions." [PATOC:GVG]
```
Comment (shawntanzk) [inline src/ontology/pato-edit.obo:15053]: change 'anatomical constituent' to 'anatomical structure'

**Review by dosumis — DISMISSED** [review]
Looks good, as long as we change constituent -> structure in definitions.

**Pull request merged**